Subtester/Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.unity3d.player"
```

Comment:
Hmm wasn't this deprecated? I think now you need to specify the package in the build.gradle right? The way it was in the `mainTemplate.gradle`... Having said that, when exporting the subtester project, it always adds it for me, and I need to remove if from the AndroidManifest manually 🥴

Reply:
Ohh right, make sense... But yeah... I guess this won't build with Android Studio so it would be great to update the Unity version... but well, if it's too much of a blocker, I think this might be an okish temporal solution.

Reply:
yeah, we are going to have to update manually, as we already had to before https://github.com/RevenueCat/purchases-unity/pull/653. Good news is that it looks like I am managing to build with the new editor in CI, so this shouldn't last for too long